controllers/services/monitoring/monitoring_controller_actions.go

```diff
@@ -128,10 +140,15 @@ func updateStatus(ctx context.Context, rr *odhtypes.ReconciliationRequest) error
 		}
 	}
 
-	m.Status.Phase = "NotReady"
 	if len(promDeployment.Items) == 1 && ready == 1 {
+		// TODO: deprecate phase
```

Comment:
the `if` before should either be rewritten like `len(promDeployment.Items) ==  ready` or there are more than the expected number of deployment should be reported as part of the failure condition

Reply:
updated